qml/Launcher/Drawer.qml

```diff
@@ -155,18 +156,21 @@ FocusScope {
 
     Rectangle {
         anchors.fill: parent
-        color: "#111111"
-        opacity: 0.99
+        color: "#BF000000"
```

Comment:
you migh want to use lowercaps color codes since thats kinda standard now.

Reply:
Can change, it just appeared by reverse-applying the Canonical patch on top of the current state.

Reply:
Yes please ;)

Reply:
I could also make color and opacity work the way it was before, since those must have been changed for a reason and are part of what is expected by the static blur parts. Ideally I just add the enabled/visible properties to it.

Reply:
Just checked again:

1) We need the "BF000000" as otherwise it would turn the transparent one black
2) For consistency across the codebase I'd argue we should keep the uppercase version, since it's uppercase everywhere